code/public/index.html

```diff
@@ -1,34 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>Technigo React App</title>
+    <!-- HTML Meta Tags -->
+    <title>Eccentric Tasks</title>
+    <meta name="description" content="A to-do list for eccentric people." />
+
+    <!-- Google / Search Engine Tags -->
+    <meta itemprop="name" content="Eccentric Tasks" />
+    <meta itemprop="description" content="A to-do list for eccentric people." />
+    <meta itemprop="image" content="https://ibb.co/HpcWnwf" />
```

Comment:
Nice touch adding these tags for SEO! Haven't tried these myself but will definitely look into it!